src/framework/theme/components/timepicker/timepicker.component.html

```diff
@@ -0,0 +1,93 @@
+<nb-card *nbPortal [class.supports-scrollbar-theming]="!isFirefox()"
+         class="nb-timepicker-container">
+  <nb-card-header class="column-header">
+    <ng-container *ngIf="!singleColumn; else fullTimeHeadersBlock">
```

Comment:
Please, remove negation from the if expression and swap if/else blocks. It would be much easier to read.